note_taking-explainer.md

```diff
@@ -17,6 +17,19 @@ new note. It wouldn't cover all the use-cases linked above (eg. voice assistant
 actually adding items to the note) but it covers the core flow of adding a new
 note.
 
+### Background
+
+Other web app APIs set a precedent of using a URL declared in the web app
+manifest to advertise capabilities and facilitate integrations with the OS:
+
+* [Web Share Target](https://w3c.github.io/web-share-target/)
+* [File Handling](https://github.com/WICG/file-handling/blob/main/explainer.md)
+* [Declarative Web Actions](https://github.com/slightlyoff/declarative_web_actions)
+
+[Protocol Handlers](
```

Comment:
Actually there's a newer proposal https://web.dev/url-protocol-handler/ for doing protocol handlers in the manifest just like the above 3. It's a lot more relevant to this than the old HTML registerProtocolHandler, so delete this, and put the above link into the list.